docs/README.md

```diff
@@ -24,101 +24,138 @@ kubectl -n gangway create secret generic gangway-key \
   --from-literal=sesssionkey=$(openssl rand -base64 32)
 ```
 
-## Docker image
+## Detailed Instructions
 
-A recent release of Gangway is available at
+The following guide is a more detailed review of how to get Gangway and other components configured.
+It's just an example of how a fully functional system can be constructed and components used may vary depending on the specific environment.
+
+It's also important to note that this guide does not include an Auth provider, that comes in the next section:
+
+We will use the following components:
+
+- [gangway](https://github.com/heptiolabs/gangway): OIDC client application
+- [contour](https://github.com/heptio/contour): Kubernetes Ingress controller
+- [cert-manager](https://github.com/jetstack/cert-manager): Controller for managing TLS certificates with Let's Encrypt.
+
+### Cert-Manager
 
+Run:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.4.1/contrib/manifests/cert-manager/with-rbac.yaml
 ```
-gcr.io/heptio-images/gangway:<tag>
+
+Apply the following config to create the staging and production `ClusterIssuer` making sure to update the `email` field below with your email address:
+
+```sh
+cat <<EOF | kubectl create -f - 
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: cert-manager
+spec:
+  acme:
+    email: <enter email here>
+    http01: {}
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    server: https://acme-v02.api.letsencrypt.org/directory
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+  namespace: cert-manager
+spec:
+  acme:
+    email: <enter email here>
+    http01: {}
+    privateKeySecretRef:
+      name: letsencrypt-staging
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+EOF
+```
+
+### Contour
+
+Run:
+
+```sh
+kubectl apply -f https://j.hept.io/contour-deployment-rbac
 ```
 
-## Config file
```

Comment:
Would it make sense to move this to a separate file, instead of deleting it? Otherwise it seems like some config options might become undocumented.

Reply:
We could, it was just a copy of the yaml, so any change would need to be duplicated every time between the two files. 

I think we'd be better off for a documentation table which outlines each configuration parameter in more detail.

Reply:
This! A table describing each option would definitely be helpful as well. Seems like we could tackle it in a follow up issue.

Reply:
I opened #52